website/docs/docs/build/incremental-overview.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "About Incremental models"
+description: "Learn about incremental models, when to use them, and how they work in dbt."
+id: "incremental-overview"
+pagination_next: "docs/build/incremental-models"
+pagination_prev: null
+---
+
+Incremental models in dbt is a [materialization](/docs/build/materializations) strategy designed to efficiently update your data warehouse tables by only transforming and loading new or changed data since the last run. Instead of processing your entire dataset every time, incremental models append or update only the new rows, significantly reducing the time and resources required for your data transformations.
+
+This overview page will provide you with a brief overview of incremental models, their importance in data transformations, and the core concepts of incremental materializations in dbt.
+
+<Lightbox src="/img/docs/building-a-dbt-project/incremental-diagram.jpg" width="60%" title="A visual representation of how incremental models work. Source: Materialization best practices page." />
+
+## Understand incremental models
+
+Incremental models enable you to significantly reduce the build time by just transforming new records. This is particularly useful for large datasets, where the cost of processing the entire dataset is high.
+
+Incremental models [require extra configuration](/docs/build/incremental-models) and are an advanced usage of dbt. You can use it when your dbt runs are becoming too slow.
```

Comment:
Should we say "too slow or costly"?
```suggestion
Incremental models [require extra configuration](/docs/build/incremental-models) and are an advanced usage of dbt. We recommend using them when your dbt runs are becoming too slow.
```

Reply:
it can be both, i think! for example, if they're model runs are taking a long time to run, it's a signal that they should think about their materializations. for example, the below dashboard in explorer lets users know their longest model-running time. so creating an incremental materialization will really reduce the build time by just transforming new records. and then help lower compute costs too.

![Screenshot 2024-03-18 at 14 48 48](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/96bfab69-69db-4cab-ae70-de8d2313a92f)